0002-lang-process.md

```diff
@@ -0,0 +1,493 @@
+
+1.  A working implementation as a `#lang`
+2.  Documentation of any and all new forms in the syntax proposed.
+3.  The following tooling
+    1.  syntax highlighting
+    2.  pretty printer
+    3.  converters to and from s-exp syntax (where relevant)
+4.  The results of measures on the dimensions of the `#lang` that are
+    determined by the community to be relevant for assessing a language
+    for inclusion. Some potential dimensions are listed in the next
+    section.
+
+All RFCs are suggested to provide the following.
+
+1.  The use cases that the language and syntax are optimized for or were
+    designed for as well as their indented audience.
+2.  An initial consideration of tradeoffs/costs/benefits of any new
+    syntax from the perspective of the proposing member(s).
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+## Dimensions
+
+A number of these dimensions are listed as a matter of completeness. In
+practice some have obvious, or trivial requirements, such as the Racket
+version that they support, which would clearly have to be at least the
+current version. Another example is the required runtime, where it seems
+fairly uncontroversial to add a restriction that the `#lang` should
+support `racketcs` and must not depend on any external runtime or
+foreign functions. Certain `#lang` s might not need to address as many
+of the points, for example if a `#lang` uses s-expression syntax then it
+would be sufficient to state that that is in line with current communal
+standards as well as in line with the current core implementation.
+
+### Runtime
+
+1.  Which Racket runtimes does the `#lang` support?
+
+    -   racketcs
+    -   racket3m
+    -   racketcgc
+    -   pycket
+
+2.  What versions of racket are supported?
+
+    This is included as a trivial case where a `#lang` that only works in
+    old versions of Racket would be ruled out.
+
+3.  Does the `#lang` depend on a runtime beyond the core Racket runtime?
+
+4.  Does the `#lang` use the FFI?
+
+### Syntax
+
+This is an incomplete list of high level properties many of which could
+be operationalized to evaluate `#lang` s for use in the core. This list
+partially overlaps the list in the
+[syntax-considerations](https://github.com/racket/racket2-rfcs/blob/master/syntax-considerations.md#considerations-list)
+doccument.
+
+1.  Homogeneity
+
+    Local structure, such as homogeneity, regularity etc. Is there
+    always local correctness of the surface syntax?
+
+2.  Regularity of visual form
+
+    This is a tough one – the primary issues is that in Racket you can
+    use \[\] and {} as a way to visually group blocks to make them stand
+    out and easier for people who read code visually (rather than
+    verbally / mathematically) to literally see what is going on.
+
+3.  Confusability (possibly fits under robustness)
+
+    How easy is it to change a single visually similar charachter and
```

Comment:
typo

Reply:
Thanks! Now I know not to ever trust default `flyspell-mode` behavior.